bootstrap.py

```diff
@@ -77,45 +80,23 @@ def region():
 
 def resource_tags():
```

Comment:
This chance cannot be merged upstream without a major version bump. It breaks the contract between `forge` and infrastructure provisioning such as `aws-infrastructure` by removing documented fallback mechanisms.